scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -11,6 +11,10 @@ WebPublisherManagerController.$inject = ['$scope', 'publisher', 'modal'];
 export function WebPublisherManagerController($scope, publisher, modal) {
     var self = this;
```

Comment:
I wouldn't use `self` if it isn't needed. But I am ok with leaving it. Just a comment.